ui/app/rif/components/item-with-actions/index.js

```diff
@@ -94,7 +96,7 @@ class ItemWithActions extends Component {
                  <FontAwesomeIcon
                    icon={faChevronRight}
                    className={''}
-                   onClick={onRightChevronClick}
+                   onClick={onRightChevronClick || onClick}
```

Comment:
I dont understand why we need to specify two possible onClick functions, why not just use one ?

Reply:
Was going to let the onRightChevronClick because maybe in some places we only want the chevron-row to be clicked. Do you think its better to only use 1 method?

Reply:
I think we should make all the row clickable always, therefore the onRightChevronClick can be removed